Satzung.tex

```diff
@@ -104,7 +106,8 @@ \section{Zweck des Vereines und Zweckverwirklichung}
 
 		\item Bereitstellung der physischen und elektronischen Infrastruktur zur
 			Durchführung von Projekten im Sinne des Satzungszwecks,
-			insbesondere die Einrichtung eines Hardwarelabors
+			insbesondere die Einrichtung eines Hardwarelabors sowie einer 
+			Werkstatt für Arbeiten aller Art, insbesondere an Holz und Metall
```

Comment:
Evt kann man das so formulieren, das es Werkstoff unabhängig ist, weil so mit Kohlenstoff zu arbeiten fände ich auch sehr cool. Also sowas in der Richtung:

> „insbesondere die Einrichtung eines Hardwarelabors sowie einer Werkstatt zur Arbeit an diversen Werkstoffen (Holz, Metal usw).“